procedures/tests/sit-stand-sit_v0.0.1.md

```diff
@@ -0,0 +1,9 @@
+# sit-stand-sit_v0.0.1
+Procedure for recording data under the environment of sitting, then standing, then sitting again
+
+## Procedure
+1) Remain still and seated for 3.5 minutes
+2) Slowly stand up (about 5 seconds) 
```

Comment:
DO you think there is any merit to add comments that highlight observations you have made during the steps?
For example, if you have experienced that standing up snags some sensor wires or you have felt lightheaded if you immediately stand up.
I think it might be a good addition, but then it dilutes the contents of the test and adds more information than strictly necessary. Curious to hear what you think about it.